Corona-Warn-App/src/main/res/drawable/circle_badge_count.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+    <size android:width="20dp" android:height="20dp"/>
+    <solid android:color="#ED382F" />
+</shape>
```

Comment:
@AlexanderAlferov  create one already for Certificate details can we re-use it ?

Reply:
Yes, tried using it but it has a stroke and it shows up as an oval. Adding a new one for this case seemed like a more elegant sollution.

Reply:
I just updated the colour to use the common one